### PR TITLE
README.md - fix some typos

### PR DESCRIPTION
**Pull request by lorand-horvath**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [x ] **typo fix**
- [ ] **metadata update**

### CODE REVIEW

**Comment by snitin315** [comment]
@lorand-horvath Can you accept CLA?

**Comment by lorand-horvath** [comment]
> @lorand-horvath Can you accept CLA?

That CLA asks me to jump through hoops... asking for signature by hand and other bs. Sorry, won't do all of that just to merge in some 2nd grade grammar fixes.

File: README.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ Supported tags and attributes:
 
 #### `boolean`
 
-The `true` value enables processing of all default elements and attributes, the `false` disable processing of all attributes.
+The `true` value enables processing of all default elements and attributes, the `false` disables processing of all attributes.
```
Comment (snitin315) [inline README.md:129]:
```suggestion
The `true` value enables the processing of all default elements and attributes, the `false` value disables the processing of all attributes.
```

**Comment by alexander-akait** [comment]
@snitin315 Due missing CLA autorizaiton feel free to resend a PR with fixes, thank you

**Comment by alexander-akait** [comment]
Fixed

**Pull request closed without merging**